pkg/controllers/updaterun/controller_integration_test.go

```diff
@@ -335,6 +335,9 @@ func generateTestClusterStagedUpdateStrategy() *placementv1beta1.ClusterStagedUp
 								Duration: time.Second * 4,
 							},
 						},
+						{
+							Type: placementv1beta1.AfterStageTaskTypeApproval,
+						},
```

Comment:
we lost the coverage that the afterStageTask has only 1 task

Reply:
Add one more test case to cover single tasks.